network.c

```diff
@@ -1145,6 +1145,17 @@ struct iio_context * network_create_context(const char *host)
 		ret = getaddrinfo(addr_str, port, &hints, &res);
 	} else {
 		ret = getaddrinfo(host, port, &hints, &res);
+#ifdef _WIN32
+		/* Yes, This is lame, but Windows is flakey with local addresses
+		 * WSANO_DATA = Valid name, no data record of requested type.
+		 * Normally when the host does not have the correct associated data
+		 * being resolved for. Just ask again. Try a max of 2.5 seconds.
+		 */
+		for (i = 0; HAVE_DNS_SD && ret == WSANO_DATA && i < 10; i++)
```

Comment:
Missing opening brace {.

Reply:
yeah - sorry - copy paste from windows to linux to git...